packages/desktop-gui/src/runs/setup-project-modal.jsx

```diff
@@ -154,55 +154,25 @@ class SetupProject extends Component {
             <a onClick={this._openOrgDocs}>
               <i className='fas fa-question-circle'></i>
             </a>
+
           </label>
+          <a
+            href='#'
+            className='btn btn-link manage-orgs-btn pull-right'
+            onClick={this._manageOrgs}>
+            Manage organizations
+          </a>
         </div>
         <div className='owner-parts'>
-          <div>
-            <div className='btn-group' data-toggle='buttons'>
-              <label className={cs('btn btn-default', {
-                'active': this.state.owner === 'me',
-              })}>
-                <input
-                  type='radio'
-                  name='owner-toggle'
-                  id='me'
-                  autoComplete='off'
-                  value='me'
-                  checked={this.state.owner === 'me'}
-                  onChange={this._updateOwner}
-                />
-                <img
-                  className='user-avatar'
-                  height='13'
-                  width='13'
-                  src={`${gravatarUrl(authStore.user && authStore.user.email)}`}
-                />
-                {' '}Me
-              </label>
-              <label className={`btn btn-default ${this.state.owner === 'org' ? 'active' : ''}`}>
-                <input
-                  type='radio'
-                  name='owner-toggle'
-                  id='org'
-                  autoComplete='off'
-                  value='org'
-                  checked={this.state.owner === 'org'}
-                  onChange={this._updateOwner}
-                />
-                <i className='far fa-building'></i>
-                {' '}An Organization
-              </label>
-            </div>
-          </div>
           <div className='select-orgs'>
-            <div className={cs({ 'hidden': this.state.owner !== 'org' || this._hasOrgsOtherThanDefault() })}>
+            <div className={this._hasOrgs() ? 'hidden' : ''}>
```

Comment:
Can we add a loading indicator for the orgs list?

I don't know if my Internet is slow or what, but the "You don't have any organizations yet" screen appeared for a brief second before I got the actual dropdown with my orgs, since there's no loading state. Users with very slow Internet might be confused when that happens unless we add a loading indicator. 

There is some prior art for "loading" in the desktop-gui in `project.jsx`, where a `<Loader>` component is used.

Reply:
The react-select supposedly supports some loading indicator also, but I didn't look into it.

Reply:
Before/After gifs of loading can be found in the main comment